app-backend/batch/src/main/scala/com/azavea/rf/batch/export/CheckExportStatus.scala

```diff
@@ -94,28 +94,29 @@ case class CheckExportStatus(exportId: UUID, statusURI: URI, time: Duration = 60
     }
   }
 
-  def notifyExportOwner(status: String) = {
+  def notifyExportOwner(status: String): Unit = {
+    logger.info(s"Preparing to notify export owners of status: ${status}")
     val export = ExportDao.query.filter(fr"id = ${exportId}").select.transact(xa).unsafeRunSync
-
-    if (export.owner == auth0Config.systemUser) {
```

Comment:
I couldn't quite figure out why this caused it to exit immediately ... I think it's safe to remove because there aren't situations where the system user performs exports (and an email to them wouldn't be necessarily bad anyways).